lib/rubocop/cop/metrics/line_length.rb

```diff
@@ -9,22 +9,86 @@ module Metrics
       # The maximum length is configurable.
       # The tab size is configured in the `IndentationWidth`
       # of the `Layout/Tab` cop.
+      #
+      # This cop has some autocorrection capabilities.
+      # It can programmatically shorten certain long lines by
+      # inserting line breaks into expressions that can be safely
+      # split across lines. These include arrays, hashes, and
+      # method calls with argument lists.
+      #
+      # If autocorrection is enabled, the following Layout cops
+      # are recommended to further format the broken lines.
+      #
+      # ClosingParenthesisIndentation
```

Comment:
I think you should mark those as unordered list for better output.

Reply:
Done. (I think!)